questionpy_server/web/_middlewares.py

```diff
@@ -0,0 +1,64 @@
+#  This file is part of the QuestionPy Server. (https://questionpy.org)
+#  The QuestionPy Server is free software released under terms of the MIT license. See LICENSE.md.
+#  (c) Technische Universität Berlin, innoCampus <info@isis.tu-berlin.de>
+from collections.abc import Iterable
+
+from aiohttp import web
+from aiohttp.typedefs import Handler, Middleware
+from aiohttp.web_request import Request
+from aiohttp.web_response import StreamResponse
+
+from questionpy_common.api.qtype import InvalidQuestionStateError
+from questionpy_common.error import QPyBaseError
+from questionpy_server.web.errors import (
+    InvalidPackageError,
+    OutOfMemoryError,
+    PackageError,
+    QpyWebError,
+    ServerError,
+    WorkerTimeoutError,
+)
+from questionpy_server.worker.exception import (
+    StaticFileSizeMismatchError,
+    WorkerCPUTimeLimitExceededError,
+    WorkerNotRunningError,
+    WorkerRealTimeLimitExceededError,
+    WorkerStartError,
+)
+from questionpy_server.worker.runtime.messages import WorkerMemoryLimitExceededError, WorkerUnknownError
+
+exception_map: dict[type[QPyBaseError], type[QpyWebError]] = {
+    InvalidQuestionStateError: InvalidPackageError,
+    StaticFileSizeMismatchError: InvalidPackageError,
+    WorkerNotRunningError: InvalidPackageError,
```

Comment:
Diese Exception deutet eher auf einen Programmierfehler hin. Sollte daher in diese map nicht aufgenommen werden.